bindings/java/doxygen/schema/db_schema_8_6.dox

```diff
@@ -62,8 +64,10 @@ Contains information about each set of images that is stored in the database.
 - **ssize** - Sector size of device in bytes
 - **tzone** - Timezone where image is from (the same format that TSK tools want as input)
 - **size** - Size of the original image (in bytes) 
-- **md5** - Hash of the image.  Currently, this is populated only if the input image is E01. 
-- **display_name** - display name of the image. 
+- **md5** - MD5 hash of the image
+- **sha1** - SHA-1 hash of the image
+- **sha256** - SHA-256 hash of the image
+- **display_name** - display name of the image
```

Comment:
display => Display

Reply:
I am going to do a big cleanup PR to keep things like this separate from the actual changes, so please wait until then.